cobalt/network/url_request_context.cc

```diff
@@ -245,9 +245,7 @@ URLRequestContext::URLRequestContext(
     using_http_cache_ = true;
 
     int max_cache_bytes = 24 * 1024 * 1024;
```

Comment:
Really tiny nit, but the first assignment of 24 MB has no effect here anymore, and can be just collapsed to 

`int max_cache_bytes = kSbMaxSystemPathCacheDirectorySize`;